integration-tests/src/test/java/oracle/weblogic/kubernetes/ItAuxDomainImplicitUpgrde.java

```diff
@@ -0,0 +1,261 @@
+
+    // create auxiliary image using imagetool command if does not exists
+    logger.info("creating auxiliary image {0}:{1} using imagetool.sh ", miiAuxiliaryImage, MII_BASIC_IMAGE_TAG);
+    createPushAuxiliaryImageWithDomainConfig(miiAuxiliaryImage, archiveList, modelList);
+
+    String auxImage = miiAuxiliaryImage + ":" + MII_BASIC_IMAGE_TAG;
+    Map<String, String> templateMap  = new HashMap();
+    templateMap.put("DOMAIN_NS", domainNamespace);
+    templateMap.put("DOMAIN_UID", domainUid);
+    templateMap.put("AUX_IMAGE", auxImage);
+    templateMap.put("BASE_IMAGE", WEBLOGIC_IMAGE_TO_USE_IN_SPEC);
+    templateMap.put("API_VERSION", "v8");
+    Path srcDomainFile = Paths.get(RESOURCE_DIR,
+        "upgrade", "auxiliary.domain.template.v8.yaml");
+    Path targetDomainFile = assertDoesNotThrow(
+        () -> generateFileFromTemplate(srcDomainFile.toString(),
+        "domain.yaml", templateMap));
+    logger.info("Generated Domain Resource file {0}", targetDomainFile);
+
+    // run kubectl to create the domain
+    logger.info("Run kubectl to create the domain");
+    CommandParams params = new CommandParams().defaults();
+    params.command("kubectl apply -f "
+            + Paths.get(WORK_DIR + "/domain.yaml").toString());
+    boolean result = Command.withParams(params).execute();
+    assertTrue(result, "Failed to create domain custom resource");
+
+    // wait for the domain to exist
+    logger.info("Checking for domain custom resource in namespace {0}", domainNamespace);
+    testUntil(
+        domainExists(domainUid, "v9", domainNamespace),
+        logger,
+        "domain {0} to be created in namespace {1}",
+        domainUid,
+        domainNamespace);
+
+    String adminServerPodName = domainUid + "-admin-server";
+    String managedServerPrefix = domainUid + "-managed-server";
+    
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed pod {0} to be ready in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodReadyAndServiceExists(managedServerPrefix + i, domainUid, domainNamespace);
+    }
+
+    StringBuffer getDomain = new StringBuffer("kubectl get domain ");
+    getDomain.append(domainUid);
+    getDomain.append(" -n " + domainNamespace);
+    getDomain.append(" -o yaml | grep compatibility-mode-operator");
+    // Get the domain in yaml format
+    ExecResult dresult = assertDoesNotThrow(
+                 () -> exec(new String(getDomain), true));
+    logger.info("Get domain command {0}", getDomain.toString());
+    logger.info("kubectl get domain returned {0}", dresult.toString());
+    assertTrue(dresult.stdout().contains("compatibility-mode-operator"), "Failed to implicitly upgrade v8 aux domain");
```

Comment:
@rjeberhard @ankedia do we need this check or checking for domain successfully created is enough?

Reply:
This check is th only way to know the implicit upgrade has happened.

Reply:
It's odd to do a contains()... Can you at least look for the specific field you want in the `kubectl get`?

Reply:
May be when all the servers are up and runtime application is verified,  it is not necessary to verify if the implicit schema upgrade has happened ?

Reply:
I think that we should verify that the schema is updated.

Reply:
Modified the tests to verify the InitContainer name in Spec/ServerPod section of domain resource schema